source/Marquee.js

```diff
@@ -1108,4 +1114,26 @@
 		style: 'overflow: hidden;'
 	});
 
+
+	moon.Marquee = {
+		_marqueeOnHoverControl: null,
+
+		setMarqueeOnHoverControl: function(oControl) {
+			this._marqueeOnHoverControl = oControl;
```

Comment:
You store a reference to a control but what happens if that control is destroyed? This would inherently keep it from being released to the GC and potentially cause other issues if you're calling methods on it after the fact.